### PR TITLE
Updated installation pl-PL language file for 3.6.1

### PR DESCRIPTION
**Pull request by yild**
#### Summary of Changes

Reviewed language strings with comparison to en-GB, translated new strings, removed unnecessary, corrected some spelling errors.
#### Testing Instructions

Simple code review.

Sorry don't know how to make multiple file changes with one PR :(

### CODE REVIEW

**Comment by infograf768** [comment]
I sent a pm to Stefan (Polish TT coordinator)
I suggest you ask him to ask the TT coordination to get you in the private forum

File: installation/language/pl-PL/pl-PL.ini [hunk 1]
```diff
@@ -115,24 +114,31 @@ INSTL_INSTALLING_EMAIL="Wysyłanie e-maila do %s"
 ;Email
 INSTL_EMAIL_SUBJECT="Szczegóły konfiguracji: %s"
 INSTL_EMAIL_HEADING="Poniżej znajdują się ustawienia konfiguracyjne nowej witryny internetowej Joomla!:"
-INSTL_EMAIL_NOT_SENT="Email nie został wysłany."
+INSTL_EMAIL_NOT_SENT="E-mail nie został wysłany."
 
 ;Complete view
 INSTL_COMPLETE_ADMINISTRATION_LOGIN_DETAILS="Szczegóły logowania dla administratora"
+; The word 'installation' should not be translated as it is a physical folder.
```
Comment (brianteeman) [inline installation/language/pl-PL/pl-PL.ini]: Please see this important note which applies to several strings

**Comment by infograf768** [comment]
Got a response from Stefan.
@bakual
The pl-PL ini file will be posted today on private forum

**Comment by Bakual** [comment]
I'm closing this PR as I have merged the files provided in the forum. See https://github.com/joomla/joomla-cms/commit/af8f341b1150d582b4d3cc891fcfecf9dbd0ab65

**Comment by yild** [comment]
Sorry to sat that, but you had marged code with missing keys, and with errors.

**Comment by infograf768** [comment]
@yild
Again, get in touch with Stefan and iron out your collaboration.

**Pull request closed without merging**